csharp/Facebook.Yoga/ios/CustomBuildAction.targets

```diff
@@ -0,0 +1,11 @@
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+	<PropertyGroup>
+	    <CompileDependsOn>CopyInNativeLib;$(CompileDependsOn)</CompileDependsOn>
+	</PropertyGroup>
+
+	<!-- The # and , in this path does not play nice with the binding project logic, so make a copy -->
+	<Target Name="CopyInNativeLib" Inputs="..\..\..\buck-out\gen\yoga#default,static\libyoga.a" Outputs="$(ProjectDir)libyoga.a">
```

Comment:
Oh right, i pulled the new PR that does the fat lib and it works, we will need to get #286 in.